redi-connect-front/src/components/organisms/PreFooter.tsx

```diff
@@ -9,8 +9,8 @@ import {
 } from 'react-bulma-components'
 import { useHistory } from 'react-router-dom'
 import Button from '../atoms/Button'
-import hello from '../../assets/images/hello.svg'
-import helloMobile from '../../assets/images/hello-mobile.svg'
+import hello from '../../assets/images/prefooter-desktop-new.svg'
```

Comment:
Why the name change? Is the "-new" sufix necessary? 
Maybe I'm missing something but you could have just kept the old names and do as less as possible changes.